tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./configs/base.tsconfig.json",
-  "include": [],
+  "include": [ ],
```

Comment:
Nitpick:
```suggestion
  "include": [],
```

Reply:
The Theia workspace preferences have "format on save" enabled and this is what the formatter gives when running Theia blueprint. If that is not correct we should either fix the preference of fix whatever bug is causing that in Theia. Part of the "self-hosting" epic?